docs/usage/dto/0-basic-use.rst

```diff
@@ -0,0 +1,75 @@
+Basic Use
+=========
+
+Here we demonstrates how to declare DTO types to your route handlers. For demonstration purposes, we assume that we
+are working with a data model ``User``, and already have two DTO types created in our application, ``UserDTO``, and
+``UserReturnDTO``.
+
+DTO layer parameters
+~~~~~~~~~~~~~~~~~~~~
+
+On every :ref:`Layer <layered-architecture>` of the Litestar application there are two parameters that control the DTOs
+that will take responsibility for the data received and returned from handlers:
+
+- ``dto``: This parameter describes the DTO that will be used to parse inbound data to be injected as the ``data``
+  keyword argument for a handler. Additionally, if no ``return_dto`` is declared on the handler, this will also be used
+  to encode the return data for the handler.
+- ``return_dto``: This parameter describes the DTO that will be used to encode data returned from the handler. If not
+  provided, the DTO described by the ``dto`` parameter is used.
+
+The object provided to both of these parameters must be a class that conforms to the
+:class:`DTOInterface <litestar.dto.interface.DTOInterface>` protocol.
+
+Defining DTOs on handlers
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``dto`` parameter
+---------------------
+
+.. literalinclude:: /examples/data_transfer_objects/the_dto_parameter.py
+    :caption: Using the ``dto`` Parameter
+    :language: python
+
+In this example, ``UserDTO`` performs decoding of client data into the ``User`` type, and encoding of the returned
+``User`` instance into a type that Litestar can encode into bytes.
+
+The ``return_dto`` parameter
+----------------------------
+
+.. literalinclude:: /examples/data_transfer_objects/the_return_dto_parameter.py
+    :caption: Using the ``return_dto`` Parameter
+    :language: python
+
+In this example, ``UserDTO`` performs decoding of client data into the ``User`` type, and ``UserReturnDTO`` is
+responsible for marshalling the ``User`` instance into a type that Litestar can encode into bytes.
```

Comment:
```suggestion
responsible for transforming the ``User`` instance into a type that Litestar can encode into bytes.
```

Maybe use a different term than "marshalling" here. I think it's good if we abstain from technical jargon where it's not really necessary. 

It may also be confusing some people since the way it's used here differs somewhat from how this term is commonly used (it does make sense in this context, but only if you're already familiar with the process that's going on).

Applies to usages in the rest of this document as well.